crates/app/tests/mod.rs

```diff
@@ -16,16 +17,24 @@ use axum::Server;
 use futures::channel::oneshot::channel;
 use reqwest::StatusCode;
 
+const TEST_DATA_DIRECTORY: &str = ".test_data";
+
 #[tokio::test]
 async fn app() {
     let lis = TcpListener::bind((Ipv4Addr::UNSPECIFIED, 0)).unwrap();
     let addr = format!("http://{}", lis.local_addr().unwrap());
 
+    let _ = remove_dir_all(TEST_DATA_DIRECTORY);
```

Comment:
No just at the beginning.

Reply:
Should they be cleaning up the test data at the end of successful tests? If a test fails then it might be useful to inspect the test data, but is it useful to keep it around if the tests pass? I also don't see much harm in keeping it around, but on balance if there's no advantage to keeping it around then it seems like we might as well clean it up. How do you feel?

Reply:
The amount of data is pretty small so it can be useful for people trying to learn about how it works. It also isn't very much data. We could delete them at the end of tests but I think it might be confusing. If we decide to store large files in the tests in the future we can delete them or use the memory store instead.